src/iocore/cache/Stripe.cc

```diff
@@ -934,6 +934,15 @@ Stripe::add_writer(CacheVC *vc)
 {
   ink_assert(vc);
   this->_write_buffer.add_bytes_pending_aggregation(vc->agg_len);
+  // [jv] An extra AGG_SIZE is added to the backlog here, but not in
```

Comment:
Please remove your name in the comment.    We have that information in the commit history and try not to have peoples names in the code.